docs/root/configuration/runtime.rst

```diff
@@ -182,9 +212,11 @@ The file system runtime provider emits some statistics in the *runtime.* namespa
   :header: Name, Type, Description
   :widths: 1, 1, 2
 
-  load_error, Counter, Total number of load attempts that resulted in an error
-  override_dir_not_exists, Counter, Total number of loads that did not use an override directory
-  override_dir_exists, Counter, Total number of loads that did use an override directory
-  load_success, Counter, Total number of load attempts that were successful
-  deprecated_feature_use, Counter, Total number of times deprecated features were used.
+  admin_overrides_active, Gauge, 1 if any admin overrides are active otherwise 0
+  deprecated_feature_use, Counter, Total number of times deprecated features were used
+  load_error, Counter, Total number of load attempts that resulted in an error in any layer
+  load_success, Counter, Total number of load attempts that were successful at all layers
   num_keys, Gauge, Number of keys currently loaded
+  num_layers, Gauge, Number of layers currently active (without loading errors)
```

Comment:
"without loading errors" concerns me

If you have three layers, A B C, and you attempt to reload B and it doesn't take, I'd assume all three original layers would still apply.  Otherwise a bad push negates config rather than being a no-op?  Maybe some comments about loading atomicity somewhere?

Reply:
Right, they all apply, but B we report "3" here, instead of "4".

Reply:
I've added a section on atomicity to docs in my pending PR, thanks for the suggestion.